docs/blog/_posts/2021-06-07-scala3.0.1-rc1-release.md

```diff
@@ -0,0 +1,169 @@
+date: 2021-06-07
+---
+
+Hello! We are happy to announce Scala 3.0.1-RC1 – the first release candidate in the post-3.0.0 era. With this release, we continue the work on making the compiler even more stable.
+
+<!--more-->
+
+# Experimental language features policy
+Research and experimentation has always been an integral part of the Scala community's culture. That's what made Scala the language it is right now. Like many things in engineering, however, it's a part of a trade-off between experimentation and stability. Experimenting is fun and insightful, but when stakes on your project are high, stability becomes a priority.
+
+Therefore, to ensure wide adoption of the language and its reliability in wide range of applications, we need to balance the two. We would like to have the room for trying new things out – but while doing so, we would like to keep in mind an ordinary Scala user who may not necessarily be interested in the bleeding edge and who would like their Scala dependencies and code to simply work. In the post-3.0.0 era, we must prioritize stability.
+
+With this release, we are introducing a restriction on [features marked as experimental](https://dotty.epfl.ch/api/scala/language$$experimental$.html). Now, it is not possible to use them from stable or `RC` releases. If you would like to experiment with them, you need to use a `NIGHTLY` version of the compiler. Nightlies are published every 24 hours, and you can find them on [Maven](https://repo1.maven.org/maven2/org/scala-lang/scala3-compiler_3/).
+
+The spirit of this policy is to make sure that effectively, no library published for Scala 3 contain experimental features. This way, what is experimental can be easily changed and is not subject to the guarantees of the wider language. And, most importantly, the changes to such features would not affect the community *in practice* – the guarantee not achievable if we just announced the policy without implementing a mechanism to enforce it.
+
+Having said that, we still encourage people to play with the experimental features from the `NIGHTLY` compier versions and discuss their findings. Without the curious and adventurous part of the community playing with the new features, there is no way of knowing what they are good for, and no way to decide whether they should be dropped or promoted to a stable feature.
+
+More about this change you can read in the PR [#12102](https://github.com/lampepfl/dotty/pull/12102).
+
+# Kind-projector work
+This release also brings extra features for the [Kind Projector](https://docs.scala-lang.org/scala3/guides/migration/plugin-kind-projector.html) migration support. First, PR [#12378](https://github.com/lampepfl/dotty/pull/12378) allows `_` as type lambda placeholder. Second, PR [#12341](https://github.com/lampepfl/dotty/pull/12341) brings support for the variance annotations on the placeholder. This work enhances the ability to cross-compile Scala 2 code that uses the Kind Projector plugin to Scala 3.
```

Comment:
/cc @neko-kai

Reply:
@anatoliykmetyuk I would add a link to the migration guide which has more details on how to to make use of it from a user perspective - https://docs.scala-lang.org/scala3/guides/migration/plugin-kind-projector.html – and a quick mention that `-Ykind-projector:underscores` option is required to enable it.